src/object.js

```diff
@@ -37,5 +37,62 @@ _.extend(Marionette.Object.prototype, Backbone.Events, {
   bindEntityEvents: Marionette.proxyBindEntityEvents,
 
   // Proxy `unbindEntityEvents` to enable unbinding view's events from another entity.
-  unbindEntityEvents: Marionette.proxyUnbindEntityEvents
+  unbindEntityEvents: Marionette.proxyUnbindEntityEvents,
+
+  //Proxy Radio message handling to enable declarative interactions with radio channels
+  _radioTypes: {
+    'radioEvents' : {
+      startMethod: 'on',
+      stopMethod: 'off'
+    },
+    'radioCommands' : {
+      startMethod: 'comply',
+      stopMethod: 'stopComplying'
+    },
+    'radioRequests' : {
+      startMethod: 'reply',
+      stopMethod: 'stopReplying'
+    }
+  },
+  //store the radio channels being used
+  _radioChannels: [],
```

Comment:
Whoops, that seems to have been typoed out of existence.  Good catch.  Will clean that up.